contracts/l1/Bridge.sol

```diff
@@ -5,20 +5,21 @@ import {IStarknetMessaging} from "./interfaces/IStarknetMessaging.sol";
 import {RayMathNoRounding} from "./libraries/math/RayMathNoRounding.sol";
 import {IAaveIncentivesController} from "./interfaces/IAaveIncentivesController.sol";
 import {IATokenWithPool} from "./interfaces/IATokenWithPool.sol";
+import {IAToken} from "./interfaces/IAToken.sol";
```

Comment:
I think not required, no? End of the day, you use it as an `IERC20` for `balanceOf()`